### PR TITLE
grammar improvements

### PR DESCRIPTION
**Pull request by JoseNoriegaa**
I made some grammar improvements and added the "translations" section.

### CODE REVIEW

File: docs/README.es.md [hunk 1]
```diff
@@ -31,6 +31,57 @@
 
 <br></br>
 
+##### Traducciones
```
Comment (mfix22) [inline docs/README.es.md]: @JoseNoriegaa we actually don't want the translations table on other pages sense it is too much to maintain. Please remove, if you will (:

File: docs/README.es.md [hunk 2]
```diff
@@ -129,7 +180,7 @@ Carbon es un proyecto de:
 
 ### Patrocinadores
 
-¿Tu empresa utiliza Carbon? Cosidera patrocinar el proyecto para financiar nuevas características, correcciones de errores y más.
+¿Tu empresa utiliza Carbon? Considera patrocinar el proyecto para financiar nuevas características, correcciones de errores y más.
```
Comment (mfix22) [inline docs/README.es.md]: Good catch! 👍

File: docs/README.es.md (hunk 1 above)
Comment (JoseNoriegaa) [reply docs/README.es.md]: I removed the translation table, the PR is up to date.

**Comment by mfix22** [comment]
Thanks @JoseNoriegaa ! 🙏

**Pull request merged**